swarm/network/stream/stream.go

```diff
@@ -68,6 +68,7 @@ type RegistryOptions struct {
 	SkipCheck       bool
 	DoSync          bool
 	DoRetrieve      bool
+	DoServeRetrieve bool
```

Comment:
please comment the fields and make a remark on their default and setting whereabouts